ci/builders/mac_host_engine.json

```diff
@@ -142,7 +162,7 @@
                         "--variant",
                         "host_release",
                         "--type",
-                        "impeller-golden"
+                        "dart,engine,impeller-golden"
```

Comment:
Maybe change the step name as well since this is now more than just the golden tests?

Reply:
Done!